doc/classes/Node3D.xml

```diff
@@ -131,91 +134,95 @@
 			<param index="2" name="up" type="Vector3" default="Vector3(0, 1, 0)" />
 			<param index="3" name="use_model_front" type="bool" default="false" />
 			<description>
-				Moves the node to the specified [param position], and then rotates the node to point toward the [param target] as per [method look_at]. Operations take place in global space.
+				Moves the node to the specified [param position], and then rotates the node to point toward the [param target] as per [method look_at]. The operation happens in global space (relative to the world).
 			</description>
 		</method>
 		<method name="orthonormalize">
 			<return type="void" />
 			<description>
-				Resets this node's transformations (like scale, skew and taper) preserving its rotation and translation by performing Gram-Schmidt orthonormalization on this node's [Transform3D].
+				Orthonormalizes this node's [member basis]. This resets this node's [member scale] to the default, but preserves the [member position] and [member rotation]. See also [method Transform3D.orthonormalized].
 			</description>
 		</method>
 		<method name="rotate">
 			<return type="void" />
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around axis, a unit [Vector3], by specified angle in radians.
+				Rotates this node's local [member transform] around the [param axis] by the given [param angle], in radians. The operation happens in local space (relative to the parent).
```

Comment:
This does rotate so-called local transform (which represents the transformation from the local to the parent coordinate system), around an axis specified in the **parent** coordinate system. So the operation does not happen in the local/object space, but in the parent space (as axis is defined in the parent space)!

> [!caution]
> "local space" != "parent space"/"relative to parent"

Please don't mix them.
If some existing docs refers to the parent space as "local", I would consider this as a bug. Are there such docs? :thinking:

I hope you haven't got merged any PRs introducing similar mixing of local/parent spaces? :upside_down_face:

---

`rotate_{x/y/z}(angle)` also rotate around parent space axis (+X/+Y/+Z), they're equivalent to `rotate(Vector3(1, 0, 0), angle)`/`rotate(Vector3(0, 1, 0), angle)`/`rotate(Vector3(0, 0, 1), angle)`.

---

BTW I often use local/global space etc. but I think it might be better if in the docs we'd refer to them explicitly as "local/global coordinate system", it should be clearer and more precise at the same time.
(What is space? Baby don't hurt me...)

Reply:
I am taking this to heart. 

> I hope you haven't got merged any PRs introducing similar mixing of local/parent spaces? 🙃

I am not sure, as aaronfranke scrutinised https://github.com/godotengine/godot/pull/87175 which was accounted for when making  https://github.com/godotengine/godot/pull/87334 , but a second look from you cannot hurt anything other than my self-esteem.